server/src/test/java/org/apache/kafka/server/config/MinInSyncReplicasConfigTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.server.config;
+
+import org.apache.kafka.common.test.ClusterInstance;
+import org.apache.kafka.common.test.api.ClusterConfig;
+import org.apache.kafka.common.test.api.ClusterTemplate;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MinInSyncReplicasConfigTest {
+
+    @ClusterTemplate("generateConfig")
+    public void testDefaultKafkaConfig(ClusterInstance cluster) {
+        assertEquals(5, cluster.brokers().get(0).logManager().initialDefaultConfig().minInSyncReplicas);
+    }
+
+    static List<ClusterConfig> generateConfig() {
+        var props = Map.of(ServerLogConfigs.MIN_IN_SYNC_REPLICAS_CONFIG, "5");
```

Comment:
I prefer a more explicit way.
```suggestion
        Map<String, String> props = Map.of(ServerLogConfigs.MIN_IN_SYNC_REPLICAS_CONFIG, "5");
```

Reply:
ditto